index.html

```diff
@@ -37,17 +39,26 @@ <h4>I’m a Software Developer</h4>
             </div>
             <div class="social-icons">
                 <ul>
-                    <li><a href="https://github.com/aradradev" target="_blank"><img src="images/github@2x.png" alt="Github social icon"></a></li>
-                    <li><a href="https://www.linkedin.com/in/abdoul-ramane-diallo-15b2a2262/" target="_blank"><img src="images/Linkedin icon@2x.png" alt="Linkedin social icon"></a></li>
-                    <li><a href="https://twitter.com/Abdoul_2023" target="_blank"><img src="images/twitter@2x.png" alt="Twitter social icon"></a></li>
-                    <li><a href=""><img src="images/medium@2x.png" alt="Medium social icon"></a></li>
+                    <a href="https://github.com/aradradev"><img src="images/github@2x.png" alt="Github social icon"></a>
+                </ul>
+                <ul>
+                    <a href="https://www.linkedin.com/in/abdoul-ramane-diallo-15b2a2262/"><img src="images/Linkedin icon@2x.png" alt="Linkedin social icon"></a>
+                </ul>
+                <ul>
+                    <a href=""><img src="images/angel list@2x.png" alt="angel social icon"></a>
+                </ul>
+                <ul>
+                    <a href="https://twitter.com/Abdoul_2023"><img src="images/twitter@2x.png" alt="Twitter social icon"></a>
+                </ul>
+                <ul>
+                    <a href=""><img src="images/medium@2x.png" alt="Medium social icon"></a>
                 </ul>
```

Comment:
- [ ] Great work on implementing the changes recommended earlier! However, It's better to wrap all the icons inside a single `ul` tag, with each icon placed in a separate `li` tag. This approach ensures that the proper semantics are followed, making it easier for various assistive devices to understand the website's structure and navigation. Moreover, it helps to improve the maintainability and scalability of the code, allowing for easier modifications in the future, without the need to change the markup structure of each icon separately. Overall, it will enhance the accessibility and usability of the website for all users.